building/build-debs/homeworld-admin-tools/src/iso.py

```diff
@@ -69,9 +69,12 @@ def gen_iso(iso_image, authorized_key, cdpack=None):
         preseeded = resource.get_resource("preseed.cfg.in")
         generated_password = util.pwgen(20)
         creation_time = datetime.datetime.now().isoformat()
+        git_hash = submodule.check_output("git", "rev-parse", "HEAD")
```

Comment:
Are you sure that `submodule` is the right thing here? I don't have that available on my system, do you mean `subprocess`?

Reply:
I do mean, subprocess. And as you can probably tell, I have not actually run this part of the build. 

When is the ISO generated in the Hyades build system?